lib/sir_handel.rb

```diff
@@ -141,6 +142,12 @@ def search
       }
     end
 
+    def with_trend(search)
```

Comment:
This could probably be put in helpers.rb, but I'll probably merge this in and add that to #35